testing/cloud_layouts/AWS/WithoutNAT/resources.yaml

```diff
@@ -19,10 +19,14 @@ spec:
   nodeTemplate:
     labels:
       node-role.deckhouse.io/system: ""
+      node-role.deckhouse.io/frontend: ""
     taints:
     - effect: NoExecute
       key: dedicated.deckhouse.io
       value: system
+    - effect: NoExecute
```

Comment:
But with this taints any system component without frontend toleration can't get to system nodes (like prom, trickster, etc)

Reply:
I did it in other way